examples/dispatcher.py

```diff
@@ -33,3 +34,7 @@ def request_work(agent):
         worker.connect(results_addr, alias='results')
         worker.connect(dispatcher_addr, alias='dispatcher')
         worker.each(0., request_work)
+
+    time.sleep(20)
+
+    ns.shutdown()
```

Comment:
Maybe:

```python
while len(ns.agents()) > 2:                                                     
    time.sleep(0.1)                                                               
                                                                                    
ns.shutdown()
```

To shutdown as soon as possible (and notice the difference between this example and the ventilator example.